django_multitenant/__init__.py

```diff
@@ -1,8 +1,11 @@
 # -*- coding: utf-8 -*-
+import django
+
 version = (2, 3, 2)
 
 __version__ = ".".join(map(str, version))
 
-default_app_config = "django_multitenant.apps.MultitenantConfig"
+if django.VERSION < (3, 2):
+    default_app_config = "django_multitenant.apps.MultitenantConfig"
```

Comment:
assuming this is correct, a comment might be called for. Why don't we have to set `default_app_config` in later versions?

As per earlier comment: lets standardize on a way we version check the difference between 3.1 and 3.2, if possible.

Reply:
I'm not sure standardizing will make this more readable. To have the same behavior you would need to prefix the condition with `not` (or have an empty if block and do this in the `else` part):

```python
if not django.VERSION >= (3, 2):
    ....
```

Reply:
After looking through the code I do realize that we are inconsistent in more ways in comparing to `django.VERSION`. I'll make a follow up PR to standardize a bit better.

Reply:
> to have the same behavior you would need to prefix the condition with not (or have an empty if block and do this in the else part):

If that is required lets not change it. I guess it was just me assuming they test the same thing, which might be completely a misunderstanding from my side. Am not super familiar with python and the tuple comparisons as you might have concluded from the earlier comment.